tst/standard/isomorph.tst

```diff
@@ -222,6 +233,151 @@ gap> T := MonogenicSemigroup(3, 1);
+      21, 5, 16, 4, 1, 14, 5, 21, 6, 24, 21, 21, 21, 24, 24, 24, 40, 39 ], 
+  [ 39, 5, 6, 40, 1, 4, 13, 15, 22, 19, 18, 23, 39, 7, 40, 8, 12, 3, 2, 9, 
+      10, 1, 4, 11, 19, 32, 30, 18, 28, 15, 25, 13, 28, 28, 28, 25, 25, 25, 
+      2, 3 ], 
+  [ 5, 39, 40, 6, 2, 3, 14, 16, 20, 21, 24, 17, 7, 39, 8, 40, 4, 11, 10, 1, 
+      2, 9, 12, 3, 31, 20, 17, 29, 16, 27, 14, 26, 27, 27, 27, 26, 26, 26, 1, 
+      4 ], [ 40, 6, 5, 39, 3, 2, 16, 14, 24, 17, 20, 21, 40, 8, 39, 7, 10, 1, 
+      4, 11, 12, 3, 2, 9, 17, 29, 31, 20, 26, 14, 27, 16, 26, 26, 26, 27, 27, 
+      27, 4, 1 ], 
+  [ 6, 40, 39, 5, 4, 1, 15, 13, 18, 23, 22, 19, 8, 40, 7, 39, 2, 9, 12, 3, 4, 
+      11, 10, 1, 30, 18, 19, 32, 13, 25, 15, 28, 25, 25, 25, 28, 28, 28, 3, 2 
+     ], [ 3, 4, 1, 2, 6, 5, 8, 7, 11, 12, 9, 10, 16, 4, 14, 1, 39, 20, 17, 
+      40, 6, 24, 21, 5, 27, 11, 10, 26, 7, 31, 8, 29, 31, 31, 31, 29, 29, 29, 
+      40, 39 ], 
+  [ 3, 4, 1, 2, 6, 5, 8, 7, 11, 12, 9, 10, 3, 15, 2, 13, 19, 39, 40, 18, 23, 
+      6, 5, 22, 12, 28, 25, 9, 32, 7, 30, 8, 32, 32, 32, 30, 30, 30, 40, 39 ],
+  [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 1, 14, 4, 16, 17, 40, 39, 20, 21, 
+      5, 6, 24, 10, 26, 27, 11, 29, 8, 31, 7, 29, 29, 29, 31, 31, 31, 39, 40 ]
+    , [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 2, 15, 3, 40, 18, 19, 39, 
+      5, 22, 23, 6, 25, 9, 12, 28, 8, 30, 7, 32, 30, 30, 30, 32, 32, 32, 39, 
+      40 ], [ 3, 4, 1, 2, 6, 5, 8, 7, 11, 12, 9, 10, 16, 15, 14, 13, 19, 20, 
+      17, 18, 23, 24, 21, 22, 27, 28, 25, 26, 32, 31, 30, 29, 38, 36, 37, 34, 
+      35, 33, 40, 39 ], 
+  [ 3, 4, 1, 2, 6, 5, 8, 7, 11, 12, 9, 10, 16, 15, 14, 13, 19, 20, 17, 18, 
+      23, 24, 21, 22, 27, 28, 25, 26, 32, 31, 30, 29, 37, 38, 36, 35, 33, 34, 
+      40, 39 ], 
+  [ 3, 4, 1, 2, 6, 5, 8, 7, 11, 12, 9, 10, 16, 15, 14, 13, 19, 20, 17, 18, 
+      23, 24, 21, 22, 27, 28, 25, 26, 32, 31, 30, 29, 36, 37, 38, 33, 34, 35, 
+      40, 39 ], 
+  [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 
+      21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 35, 33, 34, 37, 38, 36, 
+      39, 40 ], 
+  [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 
+      21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 34, 35, 33, 38, 36, 37, 
+      39, 40 ], 
+  [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 
+      21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 
+      39, 40 ], 
+  [ 2, 1, 4, 3, 39, 40, 39, 40, 1, 2, 3, 4, 2, 1, 3, 4, 6, 6, 5, 5, 39, 39, 
+      40, 40, 2, 1, 4, 3, 40, 40, 39, 39, 40, 40, 40, 39, 39, 39, 5, 6 ], 
+  [ 4, 3, 2, 1, 40, 39, 40, 39, 3, 4, 1, 2, 4, 3, 1, 2, 5, 5, 6, 6, 40, 40, 
+      39, 39, 4, 3, 2, 1, 39, 39, 40, 40, 39, 39, 39, 40, 40, 40, 6, 5 ] ]
+
+# IsomorphismSemigroups
+gap> S := FullTransformationMonoid(3);
+<full transformation monoid of degree 3>
+gap> T := AsMonoid(IsPBRMonoid, S);
+<pbr monoid of size 27, degree 3 with 3 generators>
+gap> IsomorphismSemigroups(S, T);
```

Comment:
A 'brute force' iso/inv test would be nice here, and with some of the other non-fail test examples, too.

Reply:
Done!